TMS.Net07.Homework.DaysOfWeek/TMS.Net07.Homework.DaysOfWeek/Program.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.DaysOfWeek
+{
+    class Program
+    {
+        enum daysofweek
```

Comment:
We usually use `Pascal case` for enum naming
```suggestion
        enum DaysOfWeek
```
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Reply:
done